pkgs/top-level/python-packages.nix

```diff
@@ -7416,6 +7416,8 @@ in {
 
   rxv     = callPackage ../development/python-modules/rxv     { };
 
+  maestral = callPackage ../development/python-modules/maestral { };
+
```

Comment:
please attempt to sort this, I know the condition of the file isn't amazing

Reply:
Unless I missed some hidden order, I've put it between `macfsevents` and `manifestparser` in a region that seems at least locally somewhat sorted.